go/example_test.go

```diff
@@ -42,17 +42,20 @@ func TestDremio(t *testing.T) {
 [INFO] GetSchema was successful.
 [INFO] Schema: schema:
   fields: 3
-    - EXPR$0: type=int64, nullable
-    - EXPR$1: type=int64, nullable
-    - EXPR$2: type=int64, nullable
+    - EXPR$0: type=int32, nullable
```

Comment:
Is this output from Dremio stable? Log comparison is usually unstable.

Reply:
Is there a possibility to use mocks here?

Reply:
Yeah but it requires modifying the example class further to allow for methods to be mocked. Since that seems out of the scope of this PR, I will create another ticket for those improvements and do it there. What do you think?

Reply:
> I will create another ticket for those improvements and do it there. What do you think?

Let's create a new ticket, yes, and rework all the tests there. It's weird to compare logs as a test results.

Reply:
Ticket for the tests: https://dremio.atlassian.net/browse/DX-98474 🙏